scriptmodules/helpers.sh

```diff
@@ -1223,18 +1223,24 @@ function joy2keyStart() {
 
     local params=("$@")
     if [[ "${#params[@]}" -eq 0 ]]; then
-        params=(kcub1 kcuf1 kcuu1 kcud1 0x0a 0x20 0x1b)
+        # Default button-to-keyboard mappings:
+        # * cursor keys for axis/dpad
+        # * enter, space and esc for buttons 'a', 'b' and 'x'
+        # * page up/page down for buttons 5,6 (shoulder buttons)
+        params=(kcub1 kcuf1 kcuu1 kcud1 0x0a 0x20 0x1b 0x00 kpp knp)
     fi
 
+    local joy2key="$(_joy2key_runcommand)"
```

Comment:
Although joy2key is included as part of runcommand, and maybe this should be changed
A core helper function shouldn't rely on the runcommand module.
So this function would be better integrated into the core imho